core/src/main/java/org/frankframework/mongodb/MongoDbSender.java

```diff
@@ -157,7 +157,7 @@ public void start() {
 		super.start();
 	}
 
-	private MongoClientFactory getClientFactory() {
+	private @Nonnull MongoClientFactory getClientFactory() {
```

Comment:
We moeten denk ik hier een regel voor bedenken, dat het er altijd boven staat in plaats van inline?

Reply:
Dan kunnen we dat voor de toekomst als standaard hanteren, ik ga nog even door het PR heen om het aan te passen